adapters/ix/ix_test.go

```diff
@@ -276,3 +276,80 @@ func TestMakeRequestsErrIxDiag(t *testing.T) {
 	_, errs := bidder.MakeRequests(req, nil)
 	assert.Len(t, errs, 1)
 }
+
+func TestMoveSid(t *testing.T) {
```

Comment:
You should move this test case to use JSON test framework and this is more accepted form of adapter testing. https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#adapter-code-tests